python/ray/data/iterator.py

```diff
@@ -271,7 +271,7 @@ def iter_torch_batches(
 
         This iterable yields a dictionary of column-tensors. If you are looking for
         more flexibility in the tensor conversion (e.g. casting dtypes) or the batch
-        format, try using :meth:`~ray.data.iterator.DataIterator.iter_batches` directly.
+        format, try using :py:meth:`~ray.data.DataIterator.iter_batches` directly.
```

Comment:
Why is the `:py:` needed here?

Reply:
Probably not necessary here - in this case it's equivalent to `:meth:`. I was messing around trying to get this to build correctly and must have put it in before realizing I needed to remove `.iterator`. I can remove it.

Reply:
Removed!